samples/fullduplex/Core_10/Shared/DataResponseMessage.cs

```diff
@@ -0,0 +1,7 @@
+using NServiceBus;
+using System;
+
+#region ResponseMessage
+// C# 12+ Primary constructor with record for immutable message
+public record DataResponseMessage(Guid DataId, string String) : IMessage;
```

Comment:
If you plan to switch to using record, the sample instructions also need to be updated.